tracker_bundle_test.go

```
@@ -0,0 +1,184 @@
+func requireGitForBundle(t *testing.T) {
+	t.Helper()
+	if _, err := exec.LookPath("git"); err != nil {
+		t.Skip("git not found in PATH, skipping bundle test")
+	}
+}
+
+// gitOutputBundle runs a git command in dir and returns trimmed stdout.
+func gitOutputBundle(t *testing.T, dir string, args ...string) string {
+	t.Helper()
+	cmdArgs := append([]string{"-C", dir}, args...)
+	cmd := exec.Command("git", cmdArgs...) //nolint:gosec
```

golangci-lint (2.11.4)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)